src/controller/event/handler/replication/event/event.go

```diff
@@ -15,7 +15,6 @@
 package event
 
 import "github.com/goharbor/harbor/src/pkg/reg/model"
-
```

Comment:
similar comment to this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380